### ISSUE TITLE
Provide a way to find out if focus is in dev tools

### ISSUE THREAD

**Issue by bpasero** [atom/electron#2772]
I cannot find a way currently to check if the keyboard focus is within devtools of a window or not. My use case is that I want to treat some actions from the menu specially, when focus is in dev tools.

**Comment by hokein** [comment]
There is a `devtools-focused` event in `BrowserWindow` already, but it's broken on OS X now, see #981.

**Comment by bpasero** [comment]
Ok I think this one can be closed as duplicate then.
